### PR TITLE
prfaq: minor updates

### PR DESCRIPTION
**Pull request by unknwon**
My feedback after reviewing https://github.com/sourcegraph/handbook/pull/5726

### CODE REVIEW

File: content/departments/engineering/adding-to-roadmap.md [hunk 1]
```diff
@@ -33,12 +33,12 @@ We don't use a formal/complicated framework for evaluating impact for now, in or
 
 That said, here’s a rough guideline for evaluating impact of customer-facing features:
 
-- **High**: Large ARR (>$5M ARR) or MAU (> 500 DAUs) impact, in the next 12 months. Fit with strategy. Well validated. Differentiated feature. Eg. a project like Batch Changes, Insights, Own, or adding support for a broadly-used codehost.
-- **Medium**: Large ARR (>$1M ARR) or MAU (> 100 DAUs) impact, in the next 12 months. Fit with strategy. Elements of validation.
+- **High**: Large ARR (>$5M ARR) or DAU (> 500 DAUs) impact, in the next 12 months. Fit with strategy. Well validated. Differentiated feature. Eg. a project like Batch Changes, Insights, Own, or adding support for a broadly-used codehost.
+- **Medium**: Large ARR (>$1M ARR) or DAU (> 100 DAUs) impact, in the next 12 months. Fit with strategy. Elements of validation.
```
Comment (unknwon) [inline content/departments/engineering/adding-to-roadmap.md:37]: @malomarrec - I thought we stop using MAUs altogether? Also seems like a typo by using "> X DAUs" to number MAU?

File: content/departments/engineering/adding-to-roadmap.md [hunk 2]
```diff
@@ -33,12 +33,12 @@ We don't use a formal/complicated framework for evaluating impact for now, in or
...
 - **Low**: Unknown or unvalidated impact. No clear fit with strategy.
 
 ## PR-FAQ tips
 
 - Try to keep the PR section no longer than 2 pages. FAQs can be longer.
 - Try to avoid duplicate proposals or proposals about work already on some team’s roadmap. It’s a good idea to quickly survey what’s already on the roadmap before raising a proposal.
-- Here's are two examples of good PR-FAQ: [PR-FAQ^2](https://docs.google.com/document/d/1X9j_wkKlCE9xTwRWefZaOE8OCeisQx6p6gzZTe9aQsI/edit#) and [Own PR-FAQ](https://docs.google.com/document/d/1NeokrfnZq7iLzRCIwvzZ9vhD6O3xYCv4trmu24w7m-E/edit#)
+- Here's are two examples of good PR-FAQ: [Job Fair](https://docs.google.com/document/d/1X9j_wkKlCE9xTwRWefZaOE8OCeisQx6p6gzZTe9aQsI/edit#) and [Sourcergaph Own](https://docs.google.com/document/d/1NeokrfnZq7iLzRCIwvzZ9vhD6O3xYCv4trmu24w7m-E/edit#).
```
Comment (unknwon) [inline content/departments/engineering/adding-to-roadmap.md:44]: Don't really know what "PR-FAQ^2" implies...

File: content/departments/engineering/adding-to-roadmap.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Adding to the roadmap
 
-We submit roadmap proposals in the form of Amazon-style PR-FAQs. Anybody can submit a PR-FAQ. PR-FAQ can come from the exec team, or from any teammate. PR-FAQ are reviewed and prioritised by the VPEng/Prod with input from the exec team. PMs help build, refine and evaluate the impact of PR-FAQs.
+We submit roadmap proposals in the form of [Amazon-style PR-FAQs](https://coda.io/@colin-bryar/working-backwards-how-write-an-amazon-pr-faq). Anybody can submit a PR-FAQ. PR-FAQ can come from the exec team, or from any teammate. PR-FAQ are reviewed and prioritised by the VPEng/Prod with input from the exec team. PMs help build, refine and evaluate the impact of PR-FAQs.
```
Comment (unknwon) [inline content/departments/engineering/adding-to-roadmap.md:3]: A good article that immediate solves my question of "what is PR-FAQ".

Comment (malomarrec) [reply content/departments/engineering/adding-to-roadmap.md:3]: 👍

**Review by malomarrec — APPROVED** [review]

**Pull request merged**